arelle/plugin/validate/ESEF_2022/Const.py

```diff
@@ -0,0 +1,349 @@
+'''
+Created on June 6, 2018
+Version 2022 created on September 19, 2022
+
+Filer Guidelines: https://www.esma.europa.eu/sites/default/files/library/esma32-60-254_esef_reporting_manual.pdf
+
+
+
+@author: Workiva
+(c) Copyright 2022 Workiva, All rights reserved.
+'''
+from __future__ import annotations
+import regex as re
+from typing import Any, Callable
+from arelle.ModelValue import qname
+from arelle.XbrlConst import all, notAll, hypercubeDimension, dimensionDomain, domainMember, dimensionDefault, widerNarrower
+
+browserMaxBase64ImageLength = 5242880 # 5MB
+
+esefTaxonomyNamespaceURIs = {
+    "http://xbrl.ifrs.org/taxonomy/20",
```

Comment:
Nit: Applies to every multiline call in this fine, indent is wrong for all of these

Reply:
This looks like a duplicate entry and perhaps missing the year?

Reply:
Preexisting: we're really only looking at making updates in this PR for rule changes in 2022. We'll work on cleaning up issues with the existing plugin.